code-challenges.test.js

```diff
@@ -41,6 +143,18 @@ const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
+// i will declare an array called accumulatingSum it will take in one array and then add all the values and return that value in a new array called accumulatedSumArray
```

Comment:
I would really like to see more pseudocode here.  While this gives some of the overall approach, this does not give a clear direction like your pseudocode does in other problems.